asset-transfer-basic/application-javascript/.editorconfig

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
would suggest not including the editor config

Reply:
done.